lib/workers/repository/onboarding/branch/config.ts

```diff
@@ -1,9 +1,35 @@
 import { RenovateConfig } from '../../../../config';
+import { getPreset } from '../../../../config/presets';
+import { PRESET_DEP_NOT_FOUND } from '../../../../config/presets/util';
 import { logger } from '../../../../logger';
 import { clone } from '../../../../util/clone';
 
-export function getOnboardingConfig(config: RenovateConfig): string {
-  const onboardingConfig = clone(config.onboardingConfig);
+export async function getOnboardingConfig(
+  config: RenovateConfig
+): Promise<string> {
+  let onboardingConfig = clone(config.onboardingConfig);
+
+  let organizationConfigRepoExists = false;
+  const organizationConfigPresetName = `local>${config.owner}/renovate-config`;
```

Comment:
Was there any scenario why we couldn't just use `config.repository.split('/')[0]` instead of fetching `owner` from each platform? Is there any scenario today where we think the results of that won't match with the owner returned from the platform?

Reply:
Hmm, I don't think so.

I started by just doing that split inside GitHub.. I'm not sure why, I guess I was just trying to match the way some other things were happening.

I can't think of any scenario where this would be different

Should I go back and change that, and remove all of the platform changes?

Reply:
I think that would be best for now, ie keep it simplest

Reply:
Rightio, I'll do that now :)